src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+interface TodoErrorProps {
+  todoLeft: number;
+  todos: Todo[];
+  handleClearCompleted: () => void;
+  filter: 'all' | 'active' | 'completed';
+  setFilter: (filter: 'all' | 'active' | 'completed') => void;
+}
+
+export const TodoFooter: React.FC<TodoErrorProps> = ({
+  todos,
+  handleClearCompleted,
+  filter,
+  setFilter,
+  todoLeft,
+}) => {
+  return (
+    todos.length > 0 && (
+      <footer className="todoapp__footer" data-cy="Footer">
+        <span className="todo-count" data-cy="TodosCounter">
+          {todoLeft} items left
+        </span>
+
+        <nav className="filter" data-cy="Filter">
+          <a
+            href="#/"
+            className={`filter__link ${filter === 'all' ? 'selected' : ''}`}
+            data-cy="FilterLinkAll"
+            onClick={() => setFilter('all')}
+          >
+            All
+          </a>
+
+          <a
+            href="#/active"
+            className={`filter__link ${filter === 'active' ? 'selected' : ''}`}
+            data-cy="FilterLinkActive"
+            onClick={() => setFilter('active')}
+          >
+            Active
+          </a>
+
+          <a
+            href="#/completed"
+            className={`filter__link ${filter === 'completed' ? 'selected' : ''}`}
+            data-cy="FilterLinkCompleted"
+            onClick={() => setFilter('completed')}
+          >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method